packages/@aws-cdk/core/lib/stack.ts

```diff
@@ -61,6 +45,13 @@ export interface StackProps {
    * @default {}
    */
   readonly tags?: { [key: string]: string };
+
+  /**
+   * Deployment environment to use while deploying this stack
+   *
+   * @default - Convention mode deployments if the 'aws-cdk:conventionModeDeployments' flag is set, legacy mode otherwise
```

Comment:
I believe we stopped calling this "convention mode", right?
```suggestion
   * @default - `DefaultDeploymentConfiguration` if the 'aws-cdk:conventionModeDeployments' featuyre flag is set, `LegacyDeploymentConfiguration` otherwise
```